cypress/e2e/notes-100/note-contains-link-to-github-issue.cy.js

```diff
@@ -5,7 +5,6 @@ import {
 } from '../../support/utils'
 
 
-/** {@link https://github.com/bldrs-ai/Share/issues/1056} */
```

Comment:
Why remove?  I think that's the correct story

Reply:
right, putting it back.